package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Sample App for JavaScript SDK view layer for Onfido identity verification",
   "scripts": {
     "predev": "npm run build",
-    "dev": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https",
+    "dev:webpack": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https --cert cert.pem --key key.pem",
```

Comment:
https://github.com/onfido/onfido-sdk-web-sample-app/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R24
Is it somewhat clear?